include/crm/common/output_internal.h

```diff
@@ -936,6 +936,19 @@ int pcmk__text_output_new(pcmk__output_t **out, const char *filename);
  */
 void pcmk__output_set_legacy_xml(pcmk__output_t *out);
 
+/*!
+ * \internal
+ * \brief Enable using the <list> element for lists
+ *
+ * \note This function is only used in limited places and should not be
+ * used anywhere new.  We are trying to discourage and ultimately remove
+ * uses of this style of list.
+ *
+ * @COMPAT This can be removed when the stonith_admin and crm_resource
+ * schemas can be changed
```

Comment:
which hopefully will be never

Reply:
What would be preventing us from changing the schema of those tools at a major version change?

Reply:
We could, but the point of XML output is to give script writers something they can rely on not changing